app/src/main/java/com/asafvaron/themoviedbtest/objects/MyMovie.java

```diff
@@ -0,0 +1,96 @@
+package com.asafvaron.themoviedbtest.objects;
+
+/**
+ * Created by asafvaron on 19/02/2017.
+ */
+public class MyMovie {
```

Comment:
I would like you to use Gson or Jackson as well. 
Get used to Using annotations.